app/views/glossary_terms/show.html.erb

```diff
@@ -9,7 +9,10 @@ add_tab_set(glossary_term_show_tabs(term: @glossary_term, user: @user))
 <div class="row mt-3">
   <div class="col-sm-8">
     <%= @glossary_term.description.tpl %>
-    <%= tag.p(link_to(search_tab_for(:Wikipedia, @glossary_term.name))) %>
+    <%= tag.p(
```

Comment:
@JoeCohen `search_tab_for` is a helper I rewrote. It works, but I forgot to splat it! Should be:

`<%= tag.p(link_to(*search_tab_for(:Wikipedia, @glossary_term.name))) %>`

This produces

`"<a id=\"search_link_to_Wikipedia_thing\" href=\"https://en.wikipedia.org/w/index.php?search=thing\">Wikipedia</a>"`

The reason for the tab helpers is to get a handle on the attributes that can then be be fed to nicer button helpers on the Bootstrap 5 branch. Check the helpers/tabs folder: there is one for pretty much every occasion, and it would be helpful for the refactor if we could use them consistently.

Generally speaking I'm using the word "tab" now to refer to these "link attribute arrays" that contain everything a reusable link needs to be printed in whatever format, and identified in integration tests. So a tab produces:
`[:link_text.l, url, { class: "foo", id: "fah" }]`

Reply:
Perfect! That's exactly what I was trying to do, but couldn't figure out how.
Thanks for showing me the way.

Reply:
I will merge and deploy this because the buggy current display looks so tacky.
In the the future, I'd like to improve the display so that it has something like the Observation "About this taxon" panel,
with an On MO link to a GlossaryTerm sear, and an On the web link to the Wikipedia search.
But that's for a different PR.